language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -374,16 +413,17 @@ class DottyLanguageServer extends LanguageServer
 
     val pos = sourcePosition(driver, uri, params.getPosition)
     val uriTrees = driver.openedTrees(uri)
-    val sym = Interactive.enclosingSourceSymbol(uriTrees, pos)
+    val path = Interactive.pathTo(uriTrees, pos)
+    val syms = Interactive.enclosingSourceSymbols(path, pos)
+    val includes = Include.definitions | Include.references | Include.imports
```

Comment:
I think including overriding/overriden symbols in the highlight would be useful too, so this could probably be Include.all

Reply:
Good idea, but I excluded the `linkedClass`.